src/installer.ts

```diff
@@ -1,5 +1,3 @@
-// Load tempDirectory before it gets wiped by tool-cache
-let tempDirectory = process.env['RUNNER_TEMPDIRECTORY'] || '';
```

Comment:
RUNNER_TEMPDIRECTORY isnt a thing :(

Reply:
i tested adding the action twice, and the first downloads and the second reuses the cache as expected. So not sure what the "wiped" comment used to mean.